distribution/sdk/sdk-app/pom.xml

```diff
@@ -32,6 +32,11 @@
             <artifactId>sample-mpegts-streamgenerator</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.codice.alliance.distribution</groupId>
+            <artifactId>sample-nsili-client</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
This dependency is used in the sdk-app here : https://github.com/codice/alliance/blob/master/distribution/sdk/sdk-app/src/main/resources/features.xml#L37.  It needs to be added to the pom as a dependency otherwise parallel builds will fail.